engines/director/detection_tables.h

```diff
@@ -9804,6 +9809,12 @@ static const DirectorGameDescription gameDescriptions[] = {
 	// TRAC report #14471
 	WINGAME1_l("frankiekaraoke", "", "FRANKIE TIERWELT - KARAOKE.EXE", "t:378fd2eba117bcaaa09da56e609e8c17", 2250316, Common::DE_DEU, 850),
 
+	// German: Fliegen, Flattern, Flugmaschinen
+	WINGAME1("flugmaschinen", "Windows 95", "FFF95.EXE", "1c1fa9c5797b8d2cdc04ad283d9e41d7", 1668305, 500),
```

Comment:
```suggestion
	WINGAME1_l("flugmaschinen", "Windows 95", "FFF95.EXE", "1c1fa9c5797b8d2cdc04ad283d9e41d7", 1668305, Common::DE_DEU,  500),
```
And similarly, below two entries